src/pages/ADN/main.ts

```diff
@@ -0,0 +1,86 @@
+import { pageInterface } from '../pageInterface';
+
+const style = require('!to-string-loader!css-loader!less-loader!./style.less');
```

Comment:
The build has problems with this. Add it like on the other pages.

https://github.com/MALSync/MALSync/blob/master/src/pages/Gogoanime/main.ts#L85

Reply:
Yes, I saw that, I was trying to remove some warning :/